app/helpers/proxmox_vm_config_helper.rb

```diff
@@ -153,7 +159,11 @@ def parse_typed_cpu(args, type)
       args.each_value(&:to_i)
       cpu = { cpu: cpu_flattened }
     end
-    config_typed_keys('lxc')[:cpu].each { |key| ForemanFogProxmox::HashCollection.add_and_format_element(cpu, key.to_sym, args, key) } if type == 'lxc'
+    if type == 'lxc'
```

Comment:
Could also `elsif` given before there's already a check on `type`.

Reply:
It will be fixed in https://github.com/theforeman/foreman_fog_proxmox/pull/239. This file is just rubocop safe autocorrect.